packages/interpretations/src/components/Buttons/__tests__/RedirectButton.spec.js

```diff
@@ -70,7 +69,7 @@ describe('components: Cards -> CardInfo component ', () => {
             });
 
             it('should render the correct tooltip text', () => {
-                const relativeTooltip = `View in ${itemTypeMap[dashboardItem.type].appName} app`;
+                const relativeTooltip = `View in Visualizer app`;
```

Comment:
why hard code instead of using a variable for the app name?

Reply:
Fixed.